gosnmp.go

```diff
@@ -71,20 +69,16 @@ type GoSNMP struct {
 	// Double timeout in each retry.
 	ExponentialTimeout bool
 
-	// Logger is the GoSNMP.Logger to use for debugging. If nil, debugging
-	// output will be discarded (/dev/null). For verbose logging to stdout:
-	// x.Logger = log.New(os.Stdout, "", 0)
+	// Logger is the GoSNMP.Logger to use for debugging.
+	// For verbose logging to stdout:
+	// x.Logger = NewLogger(log.New(os.Stdout, "", 0))
+	// For Release builds, you can turn off logging entirely by using the go build tag "gosnmp_nodebug" even if the logger was installed.
 	Logger Logger
 
-	// loggingEnabled is set if the Logger isn't nil, otherwise any logging calls
-	// are ignored via shortcircuit.
-	loggingEnabled bool
```

Comment:
The deprecation of this flag needs to be mentioned in the unreleased ``NOTE`` section in ``CHANGELOG.md``.
Could you as well add a ``[CHANGE]:`` entry for this pr?

Reply:
Added a description. How complete do you think it is?